doc/docs.md

```diff
@@ -914,14 +914,19 @@ There is no explicit declaration of intent, no "implements" keyword.
 
 ```v
 enum Color {
-    red green blue
+    red
+    green
+    blue
 }
 
 var color := Color.red
 // V knows that `color` is a `Color`. No need to use `color = Color.green` here.
 color = .green
 println(color) // "1"  TODO: print "green"?
 ```
+An enumeration, also referred to as enum is a simultaneous definition of a nominal enumerated type,as well as a set of constructors, that can be used to create or pattern-match values of the corresponding enumerated type.enables you to work with those values in a type-safe way within your code.
```

Comment:
What does 'nominal enumerated type' mean?
Also, what do you mean by a 'set of constructors' ?
Copying from https://doc.rust-lang.org/reference/items/enumerations.html is not nice. V's enums are not Rust's enums.

Reply:
V's enums are more like C's enums - they are just a set of numbers/integers.